docs/using/recipes.md

```diff
@@ -147,6 +147,36 @@ JupyterLab is preinstalled as a notebook extension starting in tag
 Run jupyterlab using a command such as
 `docker run -it --rm -p 8888:8888 jupyter/datascience-notebook start.sh jupyter lab`
 
+## Dask JupyterLab Extension
+
+[Dask JupyterLab Extension][https://github.com/dask/dask-labextension] provides a JupyterLab extension to manage Dask clusters, as well as embed Dask's dashboard plots directly into JupyterLab panes.
+
+```dockerfile
+# Start from a core stack version
+FROM jupyter/scipy-notebook:latest
+
+# Install the Dask dashboard
+RUN pip install dask_labextension ; \
+    jupyter labextension install -y --clean \
+    dask-labextension
+
+# Jupyter notebook
+EXPOSE 8888
```

Comment:
I think you can leave this `EXPOSE` out because it's already present in the base-notebook definition: https://github.com/jupyter/docker-stacks/blob/a644bde79fcf481f3a04652413761d1480f664ee/base-notebook/Dockerfile#L124

```suggestion
```